src/app.rs

```diff
@@ -30,7 +30,7 @@ cfg_if! {
 /// A snapshot in time of a dataset's statistics.
 ///
 /// The various fields are not saved atomically, but ought to be close.
-#[derive(Clone, Debug)]
```

Comment:
These extra instances are really used for testing.  In my tests I utilize both direct equality of Snapshots as well as equality between HashSets of Snapshots.  I guess we can conditionally switch derive macros  based on testing or platform, but it didn't seem worth it to me.  Let me know if you have a strong opinion about it though.